src/main/resources/camunda/generate_DJ_form.bpmn

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_0rn46su" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="4.12.0">
-  <bpmn:process id="GENERATE_DJ_FORM" name="Notify interim judgment claimant" isExecutable="true">
+  <bpmn:process id="GENERATE_DJ_FORM" name="Default Judgment" isExecutable="true" camunda:historyTimeToLive="P90D">
```

Comment:
shouldnt this be "Generate Default Judgment Form"?

Reply:
Its fine @velimd  - its only for search in camunda.